tasks/tinc_configure.yml

```diff
@@ -60,6 +60,7 @@
   command: awk '/^-----END RSA PUBLIC KEY-----$/'  /etc/tinc/{{ tinc_netname }}/hosts/{{ inventory_hostname | replace('.','_') | replace('-','_') }}
   changed_when: "public_key.stdout != '-----END RSA PUBLIC KEY-----'"
   register: public_key
+  ignore_errors: yes
```

Comment:
could you do `failed_when: false` instead?

Alternatively, what do you think of moving this task as a `lineinfile` with `check_mode: yes`? That might be better?

Reply:
I think `changed_when` is appropriate here; the issue is that awk fails, and then no further tasks get executed for that host. Apologies for being a little unclear.

I'll try the `lineinfile` approach, as it seems like it could be cleaner if it works :)

Reply:
Nevermind, I was misunderstanding `failed_when`. Will give that a go also.

Reply:
Thanks for the help!